### DISCUSSION TITLE
Clean and simple laptop sleeve

### DISCUSSION THREAD

**Discussion by Suzie97** [elementary/store#99] [Released Products]
Hey, I really like the new laptop sleeves but I want something more cleaner and less attention grabbing.

A subtle elementary logo at the bottom right on a black sleeve would be nice.

**Comment by Suzie97** [comment]
![image](https://user-images.githubusercontent.com/68198116/116179333-c6c6fe00-a734-11eb-8ce5-7c4640113503.png)

Something like this is also cool.

Its actually a mouse pad.

**Comment by cassidyjames** [comment]
Yeah, I'd like to make some simple/solid color laptop sleeve options. For now I've added a simple black logomark sleeve: https://store.elementary.io/products/227226074/2589282507

![image](https://user-images.githubusercontent.com/611168/116304594-f6d7c680-a75f-11eb-960c-3508127f01d9.png)

**Reply by Suzie97** [reply]
Looks sleek, but I can't find it on the store webpage, but the link provided works.

**Reply by cassidyjames** [reply]
@Suzie97 ah yeah, the home page is currently only rebuilt on demand (to prevent overloading the Printful API). We just need to trigger a rebuild and hope it doesn't bring the site down. :crossed_fingers:

**Reply by cassidyjames** [reply]
@Suzie97 this is now live from the store page as well!
